components/src/core/components/Button/button.scss

```diff
@@ -190,6 +182,7 @@
   white-space: nowrap;
   text-overflow: ellipsis;
   overflow: hidden;
+  width: 100%;
```

Comment:
Why can't this be added to the SelectInputComponent so it'll not impact any other buttons.

Reply:
Moved to the "select-input-button.scss"